src/app/components/BalanceBox/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAccount } from "~/app/context/AccountContext";
+
+function BalanceBox() {
+  const { balancesDecorated } = useAccount();
+
+  const [balance, sats] = balancesDecorated.accountBalance.split(" ");
+  return (
+    <div
+      className={
+        "w-full rounded-[6px]  h-[88px] flex flex-col items-center justify-center bg-white hover:bg-gray-50 text-black dark:bg-surface-02dp dark:text-white dark:hover:bg-surface-16dp shadow transition duration-150 "
+      }
+    >
+      <span className={" mt-[16px] mb-[8px] "}>
+        <span className="text-xl font-medium">{balance}</span>
+        <span className="text-xl font-normal m-1">{sats}</span>
+      </span>
+      <span className={"text-xl font-normal text-gray-500 mb-[16px] "}>
+        ~{balancesDecorated.fiatBalance}
```

Comment:
The fiat balance can be disabled (you can turn them of in your extension by going to Settings ➡️ Sats to Fiat) and therefore has to be optional. 

This component needs to work for both cases: with and without fiat currency.